src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -110,6 +110,13 @@ const CourseEnrollments = ({ children }) => {
     );
   }
 
+   const handleOnClose = () => {
+    // 1) setShowCancelledAssignmentsAlert(false);
+    // 2) patchLearnerContentAssignment
```

Comment:
[suggestion] Might recommend making the API integration to patch canceled assignments as a custom React hook using the `useMutation` hook from `@tanstack/react-query`, as it should help remove some boilerplate (e.g., needing to write a `useEffect`, etc.).